docs/user_guide/datasets/acs_public_coverage.rst

```diff
@@ -0,0 +1,262 @@
+.. _acs_public_coverage:
+ACSPublicCoverage
+-------------------------
+
+Introduction
+^^^^^^^^^^^^
+
+The ACSPublicCoverage dataset is one of five datasets created by Ding et al. [1]_ 
+as an improved alternative to the popular UCI Adult dataset [2]_.
+Briefly, the UCI Adult dataset is commonly used as a benchmark dataset when comparing
```

Comment:
Maybe I'm misunderstanding but I thought only ACSIncome was presented as a replacement for UCI Adult while the other datasets were just new datasets to evaluate algorithmic fairness (not direct improvements on UCI Adult?). The paper was titled Retiring Adult and thus only analyzed models trained on ACSIncome since the others have different prediction tasks and thus aren't really comparable? My initial thought is that describing UCI Adult isn't really relevant here, but I welcome other opinions :)

Reply:
That's a good point! I have clarified it as an improved fairness benchmark rather than a direct alternative. Thanks for pointing that out!